README.md

```diff
@@ -37,22 +37,37 @@ Face capture uses the webcam by default for capturing live photos of users. File
 
 In order to start integration, you will need the **API token**. You can use our [sandbox](https://documentation.onfido.com/#testing) environment to test your integration, and you will find the sandbox token inside your [Onfido Dashboard](https://onfido.com/dashboard/api/tokens).
 
-### 2. Generating a JSON Web Token
+### 2. Creating an applicant
 
-For security reasons, instead of using the API token directly in you client-side code, you will need to generate and include a short-lived JSON Web Token (JWT)\* everytime you initialise the SDK. The easiest way to generate a JWT is via the [JWT endpoint](https://documentation.onfido.com/#json-web-tokens) in the Onfido API:
+With your API token, you should create an applicant by making a request to the [create applicant endpoint](https://documentation.onfido.com/#create-applicant) from your server:
 
 ```shell
-$ curl https://api.onfido.com/v2/jwt?referrer=YOUR_WEBPAGE_URL \
-    -H 'Authorization: Token token=YOUR_API_TOKEN'
+$ curl https://api.onfido.com/v2/applicants \
+    -H 'Authorization: Token token=YOUR_API_TOKEN' \
+    -d 'first_name=John' \
+    -d 'last_name=Smith'
+```
+
+You will receive a response containing the applicant id which will be used to create a JSON Web Token.
+
+### 3. Generating an SDK token
+
+For security reasons, instead of using the API token directly in you client-side code, you will need to generate and include a short-lived JSON Web Token ([JWT](https://jwt.io/)) every time you initialise the SDK. To generate an SDK Token you should perform a request to the [SDK Token endpoint](https://documentation.onfido.com/#sdk-tokens) in the Onfido API:
+
+```shell
+$ curl https://api.onfido.com/v2/sdk_token
+  -H 'Authorization: Token token=YOUR_API_TOKEN'
+  -F 'applicant_id=YOUR_APPLICANT_ID'
+  -F 'referrer=REFERRER_PATTERN'
 ```
 
-Make a note of the `jwt` value in the response, as you will need it later on when initialising the SDK.
+Make a note of the `token` value in the response, as you will need it later on when initialising the SDK.
 
-\* Tokens expire 24 hours after creation. See [here](https://jwt.io/) for details on how JWTs work.
+\* Tokens expire 90 minutes after creation.
 
-### 3. Including/Importing the library
+### 4. Including/Importing the library
 
-#### 3.1 HTML Script Tag Include
+#### 4.1 HTML Script Tag Include
```

Comment:
Correct the points' numbers

* other`3.2 NPM style import` subpoint from 4. is still not updated

* also next point is `4. Adding basic HTML markup` when should be 5